ulog/log_benchmark_test.go

```diff
@@ -119,3 +121,25 @@ func BenchmarkUlogLiteWithFields(b *testing.B) {
 		})
 	})
 }
+
+func BenchmarkUlogSentry(b *testing.B) {
```

Comment:
Can you re-run the benchmarks, reported in the readme? It would be nice to have updated data.